packaging/deb/mkdeb.sh

```diff
@@ -40,7 +41,14 @@ pythonv=$($python --version |& grep -Po 'Python \K([23]\..)')
 
 #       I N S T A L L
 
-pip$pythonv install --pre --root $DESTDIR --prefix /usr --no-deps temboard-agent==$pep440v
+if [ -z "${FROMSOURCE-}" ] ;
```

Comment:
It seems to me that the else case will actually catch any `FROMSOURCE` value as long as it is defined, so that with `FROMSOURCE=0`, we will also build from source, which is arguably not what one would expect.